runtime/src/lib.rs

```diff
@@ -360,6 +360,15 @@ pub fn handle_mailbox_commands(drivers: &mut Drivers) -> ! {
     caliptra_drivers::report_boot_status(RtBootStatus::RtReadyForCommands.into());
     loop {
         enter_idle(drivers);
+        if drivers.mbox.cmd_busy() {
+            let reset_reason = drivers.soc_ifc.reset_reason();
+            if reset_reason == ResetReason::WarmReset {
+                let mut result = DisableAttestationCmd::execute(drivers);
+                if let Ok(mut resp) = result {
+                    Packet::copy_to_mbox(drivers, &mut resp);
+                }
+            }
+        }
```

Comment:
I think we want to do this outside of the loop so we only call disable attestation once. We should also print something to UART.

Reply:
Done